KarhooUISDK/CommonUI/AddToCalendar/AddToCalendarView.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  AddToCalendarView.swift
+//  KarhooUISDK
+//
+//  Created by Aleksander Wedrychowski on 25/11/2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+/// View Presenting add-to-calendar option. It does not have any actual logic implemented. To add event to the calendar, please handle `viewModel.onAddAction` and use `AddToCalendarWorker`.
+/// If succeeded, set `viewModel.state` to `.added`, to change design from button, to success information label.
+struct KarhooAddToCalendarView: View {
+
+    // MARK: - Nested types
+
+    private enum Constants {
+        static let height: CGFloat = 28
+        static let calendarIconSide: CGFloat = 20
```

Comment:
Can we use something from UIConstans: Dimension.Button and Dimension.Icon?

Reply:
Not really. I tried using commonly used values, and it was not looking good ;/